lib/bambora/builders/www_form_parameters.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Builders
+    ##
+    # Builds WWW URL Encoded request parameters from a Hash
+    class WWWFormParameters
```

Comment:
Would this be `WwwFormParameters`? How does the autoloader work with multiple capitalized characters in a row?

Reply:
I believe as long as you `require` the correct file, the classes in that file will be visible. The autoloader should only apply to Rails?